### PR TITLE
Acceptance fixes

### PR DESCRIPTION
**Pull request by nivemaham**
Adding rest of the fixes from UCL Acceptance test

### CODE REVIEW

File: src/app/pages/questions/components/introduction/introduction.component.html [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+  class="scroll-page"
+  style="height: calc(100% - 88px);"
+
+>
+  <div style="padding-bottom: 5px">
 <ion-item no-lines>
   <ion-icon item-right name="close-circle" (tap)="hideIntro()"></ion-icon>
 </ion-item>
-<img class="icons-big" src="assets/imgs/radar-logo.svg" alt="RADAR_Base Logo" />
+<!--<img class="icons-big" src="assets/imgs/radar-logo.svg" alt="RADAR_Base Logo" />-->
```
Comment (nivemaham) [inline src/app/pages/questions/components/introduction/introduction.component.html]: isn't this suppose to replaced by home app logo?

**Comment by peyman-mohtashami** [comment]
> @peyman-mohtashami BTE, in the home component, there is already a `showCompleted` that does the same, which we use to show the confetti animation. You can also use that instead of calculating this

Should I change it or it is ok now? I tested and it works.

File: src/app/pages/questions/components/introduction/introduction.component.html (hunk 1 above)
Comment (nivemaham) [reply src/app/pages/questions/components/introduction/introduction.component.html]: NVM. Let's just remove it

**Comment by nivemaham** [comment]
> > @peyman-mohtashami BTE, in the home component, there is already a `showCompleted` that does the same, which we use to show the confetti animation. You can also use that instead of calculating this
> 
> Should I change it or it is ok now? I tested and it works.

Hard one. Functionally it is ok. I think It is ok. Let's make a note of it and change it in Phase II if needed:) (hopefully)

File: src/app/pages/questions/components/introduction/introduction.component.html (hunk 1 above)
Comment (peyman-mohtashami) [reply src/app/pages/questions/components/introduction/introduction.component.html]: I rechecked it. They said in the ticket to remove it.

**Comment by peyman-mohtashami** [comment]
> > > @peyman-mohtashami BTE, in the home component, there is already a `showCompleted` that does the same, which we use to show the confetti animation. You can also use that instead of calculating this
> > 
> > 
> > Should I change it or it is ok now? I tested and it works.
> 
> Hard one. Functionally it is ok. I think It is ok. Let's make a note of it and change it in Phase II if needed:) (hopefully)

Ok. Sounds good.

**Pull request merged**